Apps/Inc/Tasks.h

```diff
@@ -57,6 +57,41 @@ void Task_BLE(void *p_arg);
 
 void Task_Idle(void *p_arg);
 
+/**
+ * @brief   TCBs
+ */
+extern OS_TCB FaultState_TCB;
+extern OS_TCB CriticalState_TCB;
+extern OS_TCB PetWDog_TCB;
+extern OS_TCB VoltTempMonitor_TCB;
+extern OS_TCB AmperesMonitor_TCB;
+extern OS_TCB LogInfo_TCB;
+extern OS_TCB CANBusConsumer_TCB;
+extern OS_TCB BatteryBalance_TCB;
+extern OS_TCB CLI_TCB;
+extern OS_TCB BLE_TCB;
+extern OS_TCB Idle_TCB;
+
+/**
+ * @brief   Stacks
+ */
+extern CPU_STK FaultState_Stk[TASK_FAULT_STATE_STACK_SIZE];
+extern CPU_STK CriticalState_Stk[TASK_CANBUS_CONSUMER_STACK_SIZE];
+extern CPU_STK PetWDog_Stk[TASK_PETWDOG_STACK_SIZE];
+extern CPU_STK VoltTempMonitor_Stk[TASK_VOLT_TEMP_MONITOR_STACK_SIZE];
+extern CPU_STK AmperesMonitor_Stk[TASK_AMPERES_MONITOR_STACK_SIZE];
+extern CPU_STK LogInfo_Stk[TASK_LOG_INFO_STACK_SIZE];
+extern CPU_STK CANBusConsumer_Stk[TASK_CANBUS_CONSUMER_STACK_SIZE];
+extern CPU_STK BatteryBalance_Stk[TASK_BATTERY_BALANCE_STACK_SIZE];
+extern CPU_STK CLI_Stk[TASK_CLI_STACK_SIZE];
+extern CPU_STK BLE_Stk[TASK_BLE_STACK_SIZE];
+extern CPU_STK Idle_Stk[TASK_IDLE_STACK_SIZE];
```

Comment:
Where are these definitions used? Do they really need to be declared extern here?

Reply:
No they don't have to be here, but all the task function declarations and other task information were already in Tasks.h so I tried to stay consistent with what is already present in the rtos branch.